### PR TITLE
Parent dashboard

### PR DESCRIPTION
**Pull request by aravintsamy-converse**

### CODE REVIEW

File: app/controllers/admin/parent_users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Controller responsible for handling parent user actions in the admin section.
+
 # This controller includes CRUD operations for managing parent user accounts.
 class Admin::ParentUsersController < ApplicationController
+  include AuthorizationHelper
+
+  before_action :check_admin_role, only: %i[new index create edit update destroy active_user]
```
Comment (aravindhan-converse) [inline app/controllers/admin/parent_users_controller.rb:7]: remove only from before action

File: app/controllers/standards_controller.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
+# frozen_string_literal: true
+
+# this is a controller for standards page
 class StandardsController < ApplicationController
+  include AuthorizationHelper
+  before_action :check_admin_role, only: %i[create index edit update destroy]
```
Comment (aravindhan-converse) [inline app/controllers/standards_controller.rb:6]: remove only from the before action

File: app/views/layouts/parent.html.erb
Comment (aravindhan-converse) [inline app/views/layouts/parent.html.erb:1]: are you using this layout?

File: app/views/parent/parent_notices/_notice.html.erb
Comment (aravindhan-converse) [inline app/views/parent/parent_notices/_notice.html.erb:1]: if this file is not need remove this

**Review by aravindhan-converse — CHANGES_REQUESTED** [review]

**Review by sivakumar-converse — APPROVED** [review]

**Review by aravindhan-converse — APPROVED** [review]

**Review by aravindhan-converse — APPROVED** [review]

**Review by aravindhan-converse — APPROVED** [review]

**Review by aravindhan-converse — APPROVED** [review]

**Review by sivakumar-converse — APPROVED** [review]

**Pull request merged**